src/hiero_sdk_python/tokens/token_kyc_status.py

```
@@ -5,18 +5,21 @@
 from enum import Enum
 from typing import Any
 from hiero_sdk_python.hapi.services import basic_types_pb2
 
+
 class TokenKycStatus(Enum):
     """
     KYC (Know Your Customer) status indicator:
 
-      • KYC_NOT_APPLICABLE – not applicable  
-      • GRANTED           – KYC granted  
+      • KYC_NOT_APPLICABLE – not applicable
```

Ruff (0.14.14)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)